app/lib/clients/vault/policy.rb

```diff
@@ -0,0 +1,37 @@
+module Clients
+  class Vault
+    class << self
+      def rotate_token
```

Comment:
Is it worth adding a test for rotate_token?

Reply:
well, yes!

Reply:
added a test case in `clients/vault_test.rb`